authors/apps/authentication/test_password_reset.py

```diff
@@ -0,0 +1,159 @@
+from rest_framework.test import APITestCase
+from django.contrib.auth import get_user_model
+from rest_framework import status
+from rest_framework.reverse import reverse as api_reverse
+from django.contrib.auth.tokens import PasswordResetTokenGenerator
+from .models import User
+from .password_token import generate_password_token
+
+class TestEmailSent(APITestCase):
+    """
+    test for class to send password reset link to email
+    """
+    def setUp(self):
+        """
+        set up method to test email to be sent endpoint
+        """
+        self.url = api_reverse('authentication:email_password')
+        self.register_url = api_reverse('authentication:user-registration')
+        self.user =  {
+            'user' : {
+                'username': 'kevin',
+                'email': 'koechkevin92@gmail.com',
+                'password': 'Kevin12345'
+            }
+        }
+        self.client.post(self.register_url, self.user, format="json")
+        User.is_active = True
+
+    def test_unregistered_email(self):
+        """
+        case where unregistered user tries to request a password
+        """
+        response = self.client.post(self.url, data={"email":"koechkevin@gmail.com"})
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        self.assertEqual(response.content, b'{"user": {"message": "The email provided is not registered"}}')
+
+    def test_email_field_missing(self):
```

Comment:
nice work here with the tests @koechkevin. Can we add more test for invalid email e.g. when a user enters something like `fawwaz.ali` instead `fawaz.ali@andela.com`